app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express"); // Import Express.
+const app = express(); // Create an instance of Expess library.
+const { getTopics } = require("./controllers/topics.controller"); // Import topics controller.
+const { handleCustomErrors, handle404Errors } = require("./errors"); // Import error handlers.
+
+app.use(express.json()); // import json from express library.
```

Comment:
We don't technically need this middleware function yet as we only have a get request. Please remove before you merge to main.